source/assets/sass/components/_playground.scss

```diff
@@ -450,3 +459,26 @@ $playground-base-colors: (
     height: 1.5rem;
   }
 }
+
+// ColorSwatch
+.cm-color-swatch {
+  display: inline-flex;
+
+  div {
+    background-color: var(--color);
+    height: var(--sl-size--swatch);
+    margin-inline-start: var(--sl-gutter--quarter);
+    outline: var(--sl-border--small) solid var(--sl-color--text-medium-dark);
+    outline-offset: calc(var(--sl-border--small) * -1);
+    width: var(--sl-size--swatch);
+  }
+
+  span {
+    font-family: var(--sl-font-family--emoji);
+    // Make slightly larger
+    font-size: var(--sl-font-size--small);
+    // Adjusted to accommodate larger font-size
+    line-height: 1.2;
+    margin-inline-start: var(--sl-gutter--quarter);
+  }
```

Comment:
Let's make the warning sign more of a decoration on the swatch rather than a full-size inline element:

![image](https://github.com/user-attachments/assets/0beae048-178c-4bb0-a0e5-2ea874439a28)

```suggestion
  position: relative;

  div {
    background-color: var(--color);
    height: var(--sl-size--swatch);
    margin-inline-start: var(--sl-gutter--quarter);
    outline: var(--sl-border--small) solid var(--sl-color--text-medium-dark);
    outline-offset: calc(var(--sl-border--small) * -1);
    width: var(--sl-size--swatch);
  }

  span {
    font-family: var(--sl-font-family--emoji);
    font-size: 0.5625rem;
    position: absolute;
    top: -0.3125rem;
    right: -0.3125rem;
  }
```

You could even replace the `span` with an `::after` based on a class name, although I don't have a strong opinion about whether or not that's better.